update and detail MV and DT support (#3779)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/redshift-configs.md

The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html) and refreshes them for every subsequent `dbt run` that you execute. For more information, see [Refresh Materialized Views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) in the Redshift docs.

Materialized views support the optional configuration `on_configuration_change` with the following values:

- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The `auto_refresh` action can applied without the need to rebuild the materialized view.

> - `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped

Reviewer: `continue`

Author: what do you mean by `continue` here?

```
</File>

### Limitations

> 

Reviewer 2: Same as above

Author: how's this?

```suggestion

Below are current limitations that we hope to address in a future release.
```

```
</File>

### Limitations

> Below are current limitations that we hope to address in a future release.

Reviewer 2: ```suggestion
We hope to address the following limitations in a future release.
```

#### Changing materialization from "materialized_view" to table or view

> Swapping a materialized view to be a table or view is not supported. The workaround is to manually drop the existing materialized view in the data warehouse before calling `dbt run` again.

Reviewer 2: ```suggestion
Swapping a materialized view to a table or view is not supported. You must manually drop the existing materialized view in the data warehouse before calling `dbt run` again.
```

> To illustrate, assume for a materialized view, `my_mv.sql`, has already been materialized to the underlying data platform via `dbt run`. If a user then changes the model's config to be `materialized="view"`, they will get an error. The workaround is to execute `DROP MATERIALIZE VIEW my_mv CASCASE` on the data warehouse before trying the model again.

Reviewer 2: ```suggestion
For example, assume that a view, `my_mv.sql`, has already been materialized to the underlying data platform via `dbt run`. If a user then changes the model's config to be `materialized="table"`, they will get an error. The workaround is to execute `DROP MATERIALIZE VIEW my_mv CASCASE` on the data warehouse before trying the model again.
```
